pkg/ovs/openflow/ofctrl_flow.go

```diff
@@ -19,7 +19,7 @@ type ofFlow struct {
 	// The Flow.Table field can be updated by Reset(), which can be called by
 	// ReplayFlows() when replaying the Flow to OVS. For thread safety, any access
 	// to Flow.Table should hold the replayMutex read lock.
-	ofctrl.Flow
+	*ofctrl.Flow
```

Comment:
this looks like the only controversial change potentially.

@MatthewHinton56  Could let us know why you made this change (which `go vet` warning was triggered)?

@wenyingd could you take a look and let us know if you're okay with this change?

Reply:
go vet was reporting assignment copies lock value
do to mutex copied by the assignment. 
By switching to pointer, the mutex was no longer copied by value.

Reply:
@MatthewHinton56 I am OK to change the `ofctrl.Flow` with a pointer, please go ahead.